packages/front-end/components/EventWebHooks/EventWebHookList/EventWebHookList.stories.tsx

```diff
@@ -47,8 +47,7 @@ export const Default = () => {
       enabled: false,
       url: "http://192.168.0.15:1115/events/webhooks?v=4",
       signingKey: "ewhk_6220c5592f03244c43a929850816d644",
-      // @ts-expect-error TS(2322) If you come across this, please fix it!: Type 'null' is not assignable to type 'Date'.
-      lastRunAt: null,
+      lastRunAt: getValidDate(new Date().toString()),
```

Comment:
Maybe the type is wrong because `lastRunAt` can definitely be null. It's the state it's in initially when it's created. It impacts the UI and shows a grey icon instead of a green or red one. Can we fix the type instead? Null is a valid state, though it's likely we only check for falsy.

Reply:
making the field nullable